webpack.config.js

```diff
@@ -57,5 +58,8 @@ module.exports = {
         use: ['style-loader', 'css-loader'],
       },
     ]
-  }
+  },
+  performance: {
+    hints: false,
```

Comment:
why disabling performance hint ?

Reply:
To prevent these warnings triggered by the swagger bundle:

```
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  swagger.js (1.29 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  swagger (1.29 MiB)
      swagger.js

Reply:
Disabling for the whole project and also for gitea js seems bad. Maybe filter swagger with [assetFilter](https://webpack.js.org/configuration/performance/#performanceassetfilter)

Reply:
Done

Reply:
For the record, on another run, I also saw warnings for the source map file, so I added the filter on the `.js` file extension:

```
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  index.js.map (455 KiB)
  swagger.js.map (4.61 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  index (583 KiB)
      index.js
      index.js.map
  swagger (4.61 MiB)
      swagger.js.map